sources/packages/api/src/route-controllers/application/application.student.controller.ts

```diff
@@ -287,8 +354,10 @@ export class ApplicationController extends BaseController {
    */
   @CheckSinValidation()
   @CheckRestrictions()
-  @AllowAuthorizedParty(AuthorizedParties.student)
   @Patch(":applicationId/draft")
+  @ApiOkResponse({ description: "Draft application updated." })
+  @ApiNotFoundResponse({ description: "APPLICATION_DRAFT_NOT_FOUND." })
+  @ApiInternalServerErrorResponse({ description: "Unexpected error" })
```

Comment:
I would say that the 500 are the implicit ones that we do not need to expose unless there is a specific comment to be added.
The 500 will be generated in the body or any time that an exception happens in the controller body.

Reply:
I added 500 to his controller, as the controller itself was throwing the internal server error 
![image](https://user-images.githubusercontent.com/77353155/158898005-fab14120-1e93-4796-a38d-4e012a08707b.png)

Reply:
Any `throw error` or unexpected exception from the body will cause the same result, my point was to add it when we need a more detailed description other than just _Unexpected error_.